sdk/Cargo.toml

```diff
@@ -66,6 +66,7 @@ fast-xml = "0.23.1"
 hex = "0.4.3"
 img-parts = "0.3.0"
 log = "0.4.8"
+lopdf = "0.30.0"
```

Comment:
IMHO we should gate these dependencies on a "pdf" crate feature.

Reply:
That's a good idea - will do